galette/lib/Galette/Core/Galette.php

```diff
@@ -758,7 +758,8 @@ public static function getListActions(Adherent $member): array
                             'id' => $member->id
                         ]
                     ],
-                    'icon' => 'user times red'
+                    'icon' => 'user times red',
+                    'ajax' => 'delete'
```

Comment:
I did something similar on my list branch; I just wonder why you call that parameter "ajax"?

Reply:
(on my side, I called it `extra_class` ;) - that's not blocking for merge)